test/e2e/atlas/access_lists_test.go

```diff
@@ -55,11 +54,10 @@ func TestAccessList(t *testing.T) {
 			"-o=json")
 		cmd.Env = os.Environ()
 		resp, err := cmd.CombinedOutput()
-		req.NoError(err)
+		require.NoError(t, err)
```

Comment:
don't you delete req in line 38?

Reply:
it's used outside of the anon function for `t.Run`, the main focus here was to avoid relying on `require` or `assert` from the closure, which then the linter was complaining